driver/jvm_tooling.cpp

```diff
@@ -115,6 +115,37 @@ constexpr const char kExceptionUtilsClassName[] =
 
 namespace jazzer {
 
+void DumpJvmStackTraces() {
+  JavaVM *vm;
+  jsize num_vms;
+  JNI_GetCreatedJavaVMs(&vm, 1, &num_vms);
+  if (num_vms != 1) {
+    return;
+  }
+  JNIEnv *env = nullptr;
+  if (vm->AttachCurrentThread(reinterpret_cast<void **>(&env), nullptr) !=
+      JNI_OK) {
+    return;
+  }
+  jclass thread = env->FindClass(kExceptionUtilsClassName);
```

Comment:
Minor: What do you think about renaming `thread` to `exceptionUtils`?

Reply:
Thanks, that was just a leftover from a previous version.